templates/data/mysql/support.html

```diff
@@ -0,0 +1,636 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Get support for MySQL, one of the most popular DBMS.{% endblock %}
+
+{% block meta_description %}
+  Get support for MySQL from Canonical, the experts in open source. We provide ticket, phone and screen sharing support for up to ten years per release track.
+{% endblock %}
+
+{% block meta_keywords %}MySQL, support{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row--25-75-on-large">
+      <div class="col">
+        <div class="p-section--shallow u-hide--medium">
+          {{ image(url="https://assets.ubuntu.com/v1/c099a85d-mysql-logo.png",
+                    alt="MySQL logo",
+                    width="852",
+                    height="243",
+                    hi_def=True,
+                    loading="auto") | safe
+          }}
+        </div>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h1>
+            Comprehensive MySQL support,
+            <br class="u-hide--small" />
+            from open source experts
+          </h1>
+          <p>
+            Our SLA-backed support options range from security maintenance for your debs, snaps and containers up to Managed MySQL on your private infrastructure or your cloud tenants.
+          </p>
+          <p>
+            Our support goes beyond MySQL server to cover the extensions and tools you typically install with MySQL, such as MySQL router, xtrabackup and more.
+          </p>
+        </div>
+        <div class="p-cta-block u-no-padding--bottom">
+          <a class="p-button--positive"
+             aria-controls="data-relational-dbs-modal"
+             href="/data/mysql#get-in-touch">Contact us</a>
```

Comment:
```suggestion
          <a class="p-button--positive"
             aria-controls="data-relational-dbs-modal"
             href="/contact-us">Contact us</a>
```
The #get-in-touch will not work if JS is disabled and in that case we want the user to reach to contact-us page